examples/plot_hf_hub.py

```diff
@@ -64,11 +65,11 @@
 # that, we need to first store the model as a pickle file and pass it to the
 # hub tools.
 
-_, pkl_name = mkstemp(prefix="skops")
+_, pkl_name = mkstemp(prefix="skops-", suffix=".pkl")
```

Comment:
Maybe that would be nice to have a note somewhere saying saving as `.pkl` is not the only way 😅 just saying

Reply:
Done.